.github/workflows/playwright.yml

```diff
@@ -5,9 +5,16 @@ jobs:
   test:
     timeout-minutes: 15
     runs-on: [self-hosted, asg]
-    # Certs added for the self hosted runner
-    env:
-      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
+    container:
+      image: public.ecr.aws/docker/library/node:18.19.0-bullseye
+      # Certs added for the self hosted runner
+      env:
+        NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
+      ports:
+        - 80
+      volumes:
+        - my_docker_volume:/home/runner
```

Comment:
where does this volume dir come from?

Reply:
I actually pulled this from another example. I think this can be safely removed.